[Doc] Contributor documentation (#109)
pasqal-io/quantum-evolution-kernel: docs/contributing/how to contribute.md

# How to contribute

We're grateful for your interest in contributing to this library!

## Asking questions

> If you have any question aout this library, don't hesitate to ask!

Reviewer: ```suggestion
If you have any question about this library, don't hesitate to ask!
```

typo

> If you have any question on this library, don't hesitate to ask them.

Reviewer 2: ```suggestion
If you have any question about this library, feel free to ask them.
```

Maybe you can suggest the Discussion tab in GitHub (if enabled) or the Slack channel ?

Reviewer 2: Ah well, I should have read next.

Author: We've decided to not create a Discussion tab in GitHub just yet, as we don't want to fragment communications too much. If and when we get hundreds and thousands of users, we'll probably have to revise this strategy :)
